src/pages/Youth/Youth.tsx

```diff
@@ -45,20 +44,27 @@ const Youth: React.FC = () => {
 
       return (
         youth?.map((item) =>
-        <div className="youth-card-container">
+        <Flex
+              dir={"row"}
+              align={"stretch"}
+              justify={"center"}
+              wrap={"wrap"}
+            >
           <DisplayYouth
             key={item.firebaseUID}
             name={item.lastName + " " + item.lastName}
             email={item.email} />
-          </div>
+          </Flex>
```

Comment:
why are you wrapping each component in its own Flex tag? did you mean to put all the youth components in one flex container?

Reply:
That is correct, will be fixed in next push